background-concealment-demo-py/BackgroundConcealmentDemo.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5-7.
g_nn_ins = cv.GInferInputs()
g_nn_ins.setInput('input', g_in)  
g_nn_outs = cv.gapi.infer(nn_id, g_nn_ins)
</pre><p> </p><p>As an idea - could these three lines be combined? If <code>setInput()</code> retuned the object</p><pre class="ql-syntax" spellcheck="false">g_nn_outs = cv.gapi.infer(nn_id, cv.GInferInputs().setInput('input', g_in))
</pre><p>I think, it's not too many reasons to keep <code>g_nn_ins</code> explicitly, if it's not necessary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TolyaTalamanov/g-api-collection/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>